### PR TITLE
Modified windows check

### PR DESCRIPTION
**Pull request by kakron**
Added part for windows 10 and modified recommendation for windows XP

### CODE REVIEW

File: en/exercises/device_behaviour_check/device_assessment_checklist.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+   * (CLI) deb: 
+```dpkg-query -l | grep virus``` rpm: ```yum list installed | grep virus```
+   * See also: https://en.wikipedia.org/wiki/Linux_malware#Anti-virus_applications
+ * User privilege
+   * (CLI) 
+```groups```
+
+ * Drive Encryption
+   * 
+   * (TrueCrypt)
```
Comment (dlshad) [inline en/exercises/device_behaviour_check/device_assessment_checklist.md]: I know this is an assessment, but there is no TrueCrypt anymore, similar to Windows XP let's change this to (VeraCrypt) or the default distro-specific full desk encryption like LUKS. See the following: https://help.ubuntu.com/community/ManualFullSystemEncryption

File: en/exercises/device_behaviour_check/device_assessment_checklist.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+   * (GUI) Control Panel -- All Control Panel Items -- Windows Firewall
+   * (CLI) 
+```Netsh Advfirewall show allprofiles```
+ * Anti-Virus Version
+   * 
+ * User privilege
+   * (GUI) Control Panel -- All Control Panel Items -- User Accounts and checking also the User Account Control settings.
+ * Drive Encryption
+   * (GUI) Control Panel -- All Control Panel Items -- BitLocker Drive Encryption
+   * (TrueCrypt) , https://diskcryptor.net/wiki/Main_Page
```
Comment (dlshad) [inline en/exercises/device_behaviour_check/device_assessment_checklist.md]: Let's remove TrueCrypt and add VeraCrypt

**Comment by joncamfield** [comment]
OK, I'm working to clean out a few lost mva index files.

It looks like you've added the outputs section to a few of the overview index files - I'd like to keep that standardized across all the overview files.  We'd cropped those down originally to get them to reliably all fit on one page printed, but that's not necessarily a priority anymore.  I opened an issue to discuss this: https://github.com/SAFETAG/SAFETAG/issues/330

We seem to have a dupe of device_checklist with device_behavior_checklist, but I don't see any updates to index/linking files, so I propose copying the updated content fro _behavior_ into the existing folder.

I want to do more in-depth review of the threat assessment checklist and its framing, as well as the work to build the org device method - thanks for tackling these!

We seem to have some conflicts which I'll look to manage.

**Comment by joncamfield** [comment]
I have merged these changes (with some edits, see above and a few commits to master) -- thanks!

**Pull request closed without merging**